config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "process.env.DB_USER",
```

Comment:
The value for `username` should be accessed using `process.env.DB_USER` without quotes to correctly retrieve the environment variable.